docs/book/introduction.md

```diff
@@ -16,15 +16,135 @@ description: Welcome to ZenML!
 
 If you answered yes to any of the questions above, ZenML is here to help you:
 
-* **ZenML** is an **open-source MLOps framework** built for data scientists, ML Engineers, and MLOps Developers.
-* It comes equipped with a simple and flexible **Python SDK** that helps you to create **portable and production-ready pipelines**.
-* It also features **cloud-** and **tool-agnostic** **interfaces/abstractions** such as orchestration systems, artifact repositories, and model deployment platforms that cover the entire ML journey.
-* With these abstractions, it remains **extensible** and **open to customization**. That is also backed by numerous **collaborative features** that allow you to work not just with your team but also with the community.
-* Moreover, this creates a separation between the different aspects of ML workflows, allowing users to focus on creating their own respective solutions in an efficient manner.
-* Overall, it reduces the complexity of your development lifecycle, all the way **from an experimentation environment to a production-grade solution**.
+**ZenML** is an extensible, open-source MLOps framework for creating portable, 
+production-ready MLOps pipelines. 
+
+By decoupling infrastructure from code, ZenML gives all ML developers in your 
+organization more freedom and independence in how they approach their work:
+
+![ZenML Overview](.gitbook/assets/intro_zenml_overview.png)
+
+{% tabs %}
+
+{% tab title="ZenML for Data Scientists" %}
+
+As a Data Scientists, ZenML gives you the freedom to focus on modeling and
+experimentation without having to worry about production issues at all.
+
+**Tooling Freedom:** Since ZenML code runs in any environment, you can 
```

Comment:
Maybe also say here (either add it or instead of ) that we help you create production-ready code *locally*  first without infrastructure worries and give you a way to structure your code properly. These are the values ive heard more often than tooling freedom

Reply:
Running locally was already mentioned before, but more as a side note. I changed the header to **Develop Locally:** now to emphasize this a bit more.